### ISSUE TITLE
Get indentation rules to work

### ISSUE THREAD

**Issue by williamspiro** [HubSpot/HubL-Language-Extension#14]
For some reason this does not work:
```
    "indentationRules": {
        "increaseIndentPattern": ".*{%(?!.*end).*%}.*",
        "decreaseIndentPattern": ".*{%(.*?end).*%}*."
    }
```
https://code.visualstudio.com/api/language-extensions/language-configuration-guide#indentation-rules

**Comment by anthmatic** [comment]
I spent some time looking at this today. It looks like this doesn't work "decoratively", most likely because we're dealing with "embedded languages". PR incoming shortly to add this programmatically.

**Comment by anthmatic** [comment]
Closed in #43
